clipmenu

```diff
@@ -57,11 +57,25 @@ else
 fi
 
 [[ $chosen_line ]] || exit 1
-file=$cache_dir/$(cksum <<< "$chosen_line")
-[[ -f "$file" ]] || exit 2
+
+files=()
+while IFS= read -r line; do
+    file=$cache_dir/$(cksum <<< "$line")
+    files+=("$file")
+done <<< "$chosen_line"
+
+paste=()
+for file in "${files[@]}"; do
+    contents=$(cat "$file")
+    paste+=("$contents")
```

Comment:
Why do all this, why not actually use `cat` to concat and send it directly to xsel?

Reply:
I tried with cat but it didn't work, I'll have another go as it would simplify the code a lot.